pkg/cfn/runner_test.go

```diff
@@ -44,8 +45,10 @@ func TestCreate(t *testing.T) {
 		t.Run(tc.name, func(t *testing.T) {
 			err := tc.runner.CreateIfNotExists(mock.DefaultStackName, []byte{}, nil, 10)
 			if tc.expectError {
+				result, _ := json.Marshal(err)
```

Comment:
You perhaps explained this before, but why are we converting to json, can we not just create an expected error struct with Mishudark?

Reply:
Might work. I can give it another try. Reason why I did this is because I'm not proficient enough in Go to sufficiently compare the two.

The reason why I converted it to JSON is because that way I could see all the information the mishudark error contained.

Reply:
I see, that makes sense. :+1: